### PR TITLE
Husholdninger123barn

### PR DESCRIPTION
**Pull request by espegun**

### CODE REVIEW

File: functions/husholdning123barn.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+import numpy as np
+import common.aws
+import common.aggregate_dfs  # as aggregate
+import common.transform  # as transform
+from common.aggregateV2 import Aggregate
+from common.templates import TemplateA
+from common.output import Output, Metadata
+from common.util import get_latest_edition_of
+
+os.environ["METADATA_API_URL"] = ""
```
Comment (aulonm) [inline functions/husholdning123barn.py]: not needed anymore

File: functions/husholdning123barn.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+s3_bucket = "ok-origo-dataplatform-dev"
```
Comment (aulonm) [inline functions/husholdning123barn.py]: not needed anymore

File: functions/husholdning123barn.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+if __name__ == "__main__":
+
+    input_data = get_latest_edition_of("husholdninger-med-barn")
+
+    # Test writing one set
+    handler(
+        {
+            "input": {"husholdninger-med-barn": input_data},
+            "output": "intermediate/green/husholdninger-med-1-barn/version=1/edition=20190520T114926/",
+            "config": {"output_set": "husholdninger-med-1-barn"},
```
Comment (aulonm) [inline functions/husholdning123barn.py]: not completely sure, but the pipeline sends object with type and not output_set. Not sure if this will crash

Comment (espegun) [reply functions/husholdning123barn.py]: Changed from "output_set" to "type" as discussed. This still requires the name of the output_set/type to be defined in the Kotlin code.

File: functions/husholdning123barn.py (hunk 2 above)
Comment (espegun) [reply functions/husholdning123barn.py]: Done.

File: functions/husholdning123barn.py (hunk 1 above)
Comment (espegun) [reply functions/husholdning123barn.py]: Done.

**Pull request merged**